third_party/move/move-compiler-v2/tests/bytecode-generator/control_flow/break_continue_outside_loop.exp

```diff
@@ -0,0 +1,45 @@
+
+Diagnostics:
+warning: Unnecessary trailing semicolon
+  ┌─ tests/bytecode-generator/break_continue_outside_loop.move:3:7
+  │
+3 │     break;
+  │     -----^
+  │     │    │
+  │     │    A trailing `;` in an expression block implicitly adds a `()` value expression after the semicolon, not needed here.
+  │     Any code after this expression will not be reached
```

Comment:
Hmm. I'm actually not convinced we need that warning. This seems to me an artifact of the compiler regards the inserted `()` which will be really hard to understand for users. Does v1 have it? If not, this will be a breaking change. (New warnings are breaking changes.)

Reply:
V1 has this warning, and I've winnowed it down so V2 produces a similar warning in almost exactly the same cases.